examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/ipex/requirements.txt

```diff
@@ -0,0 +1,13 @@
+accelerate
+protobuf
+sentencepiece != 0.1.92
+datasets >= 1.1.3
+torch >= 1.10
+transformers
+pytest
+wandb
+einops
+neural-compressor
+intel-extension-for-transformers
+git+https://github.com/EleutherAI/lm-evaluation-harness.git@83dbfbf6070324f3e5872f63e49d49ff7ef4c9b3
```

Comment:
this version is outdated, we have observed significant changes in some results with the latest version.